Firmware build fails on openSUSE Leap 15.3 after Toolchain Splitting

### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Build system

### Current Behavior

Firmware build fails after the toolchain splitting on openSUSE and probably other 64 bit Linux systems.  OpenSUSE  that use /usr/lib64 due to find_clang.py not finding libclang in the default /usr/lib64 location.

ERROR: libclang not found!
make[7]: *** [radio/src/CMakeFiles/datacopy.dir/build.make:79: radio/src/datacopy.cpp] Error 255


### Expected Behavior

Firmware to build.

### Steps To Reproduce

 Attempt to build firmware on openSUSE

### Version

Other (Please specify below)

### Transmitter

Radiomaster TX16S

### Anything else?

I will create a PR to find libclang on openSUSE and other systems that use /usr/lib64

--- a/radio/util/find_clang.py
+++ b/radio/util/find_clang.py
@@ -67,7 +67,8 @@ def findLibClang():
             "/usr/lib/llvm-6.0/lib",
             "/usr/lib/llvm-3.8/lib",
             "/usr/local/lib",
-            "/usr/lib"
+            "/usr/lib",
+            "/usr/lib64"
         ]
         libSuffix = ".so"
     elif sys.platform == "win32" or sys.platform == "msys":

